requirements/base.in

```diff
@@ -1,3 +1,4 @@
 open-api-framework
+git+https://github.com/maykinmedia/commonground-api-common.git@issue/44-migration-error
```

Comment:
Like the comment above, adding a reminder here that this should be pinned to a release version. We probably also want to get rid of pinning `commonground-api-common` in all the projects and pulling that dependency in through OAF, [see discussion](https://github.com/maykinmedia/open-api-framework/issues/66#issuecomment-2500098180).

Reply:
See https://github.com/maykinmedia/open-api-framework/issues/90